pom.xml

```diff
@@ -2023,6 +2023,46 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                    <groupId>com.github.ekryd.sortpom</groupId>
+                    <artifactId>sortpom-maven-plugin</artifactId>
+                    <version>3.2.1</version>
+                    <executions>
+                        <execution>
+                            <id>sortpom-warn</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>verify</goal>
+                            </goals>
+                            <configuration>
+                                <verifyFail>Warn</verifyFail>
+                            </configuration>
+                        </execution>
+                        <execution>
+                            <id>sortpom-fail</id>
+                            <phase>verify</phase>
```

Comment:
Remove this. The default phase is `validate` which is the correct phase, since it runs at the start of the build, rather than `verify` which runs at the end and is intended to verify the project artifacts (rather than `validate` which is to validate the project setup).

Reply:
Warn will report problem and Stop will fail build. Unfortunately Stop was not reporting error :( . But I will check it again.

Reply:
This was done to enforce execution of sortpom after enforcer pedantic rule.
But once it's gone I will move it to validate.